[2024-05-01T18:41:31.199+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-01T18:41:31.228+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_pipeline.read_and_clean_csv manual__2024-05-01T17:58:25.196953+00:00 [queued]>
[2024-05-01T18:41:31.243+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_pipeline.read_and_clean_csv manual__2024-05-01T17:58:25.196953+00:00 [queued]>
[2024-05-01T18:41:31.245+0000] {taskinstance.py:2303} INFO - Starting attempt 5 of 10
[2024-05-01T18:41:31.267+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): read_and_clean_csv> on 2024-05-01 17:58:25.196953+00:00
[2024-05-01T18:41:31.277+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=942) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-01T18:41:31.279+0000] {standard_task_runner.py:63} INFO - Started process 944 to run task
[2024-05-01T18:41:31.279+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'weather_data_pipeline', 'read_and_clean_csv', 'manual__2024-05-01T17:58:25.196953+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/demo.py', '--cfg-path', '/tmp/tmphxgm6xum']
[2024-05-01T18:41:31.281+0000] {standard_task_runner.py:91} INFO - Job 10: Subtask read_and_clean_csv
[2024-05-01T18:41:31.347+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_data_pipeline.read_and_clean_csv manual__2024-05-01T17:58:25.196953+00:00 [running]> on host a747c391e266
[2024-05-01T18:41:31.444+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data_pipeline' AIRFLOW_CTX_TASK_ID='read_and_clean_csv' AIRFLOW_CTX_EXECUTION_DATE='2024-05-01T17:58:25.196953+00:00' AIRFLOW_CTX_TRY_NUMBER='5' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-01T17:58:25.196953+00:00'
[2024-05-01T18:41:31.446+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-01T18:41:31.699+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-01T18:41:31.701+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-01T18:41:31.713+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=weather_data_pipeline, task_id=read_and_clean_csv, execution_date=20240501T175825, start_date=20240501T184131, end_date=20240501T184131
[2024-05-01T18:41:31.774+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-01T18:41:31.801+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-01T18:41:31.804+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
